conda_recipe_manager/fetcher/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""
+:Description: Module that provides utilities for acquiring remote resources.
+"""
+
+import logging
+
+# Default to emitting no logs. It is up to the client program to define logging conditions.
+logging.getLogger(__name__).addHandler(logging.NullHandler())
```

Comment:
You might want to add a Null handler from `conda_recipe_manager.__init__` and save yourself from ahving to set one in each sub level.